Description: this patch disables the cap on oslo.db, oslo.messaging and eventlet.
 Also removes the restriction on SQLAlchemy as these versions were not shipped on Debian.
 .
Origin: vendor
Forwarded: not-needed
Reviewed-By: gustavo panizzo <gfa@zumbi.com.ar>
Last-Update: <2015-02-06>

--- a/requirements.txt
+++ b/requirements.txt
@@ -9,7 +9,7 @@
 anyjson>=0.3.3
 argparse
 Babel>=1.3
-eventlet>=0.15.1,<0.16.0
+eventlet>=0.15.1
 greenlet>=0.3.2
 httplib2>=0.7.5
 requests>=1.2.1,!=2.4.0
@@ -19,15 +19,15 @@
 keystonemiddleware>=1.0.0
 netaddr>=0.7.12
 python-neutronclient>=2.3.6,<3
-SQLAlchemy>=0.8.4,<=0.9.99,!=0.9.0,!=0.9.1,!=0.9.2,!=0.9.3,!=0.9.4,!=0.9.5,!=0.9.6
+SQLAlchemy>=0.8.4
 WebOb>=1.2.3
 python-keystoneclient>=0.10.0
 alembic>=0.6.4
 six>=1.7.0
 stevedore>=1.0.0  # Apache-2.0
 oslo.config>=1.4.0  # Apache-2.0
-oslo.db>=1.0.0,<1.1  # Apache-2.0
-oslo.messaging>=1.4.0,!=1.5.0,<1.6.0
+oslo.db>=1.0.0
+oslo.messaging>=1.4.0
 oslo.rootwrap>=1.3.0
 
 python-novaclient>=2.18.0
